src/popup/popup.js

```diff
@@ -279,6 +279,37 @@ elements.searchIcon.addEventListener('click', () => {
 
       pageNo += 1;
     });
+  elements.clearSearchButton[0].removeAttribute('style');
+});
+
+elements.clearSearchButton[0].addEventListener('click', () => {
+  // Remove Old
+  elements.clearSearchButton[0].setAttribute('style', 'display: none');
+  elements.inputField.value = '';
+  removeLoaderAnimation();
+  removeNode('no-image-found');
+  removeOldSearchResults();
+  removeLoadMoreButton(elements.loadMoreButtonWrapper);
+  elements.gridPrimary.setAttribute('style', 'position: relative; height: 0px;');
+  localStorage.clear();
+
+  // Bring Initial Content
```

Comment:
Please use the already defined function `restoreInitialContent(context)` (defined [in the utils](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/utils.js#L24))